src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -105,12 +144,27 @@ def calculate_timeout_seconds(
     timeout_default=pytest.options.timeout_default,
     timeout_maximum=pytest.options.timeout_maximum,
   )
+
+  coverage_args = []
+  if pytest.options.coverage:
+    packages_to_cover = get_packages_to_cover(
```

Comment:
Note that the `python_tests` target type has a `coverage` argument that should probably be consumed somewhere in here. See https://www.pantsbuild.org/build_dictionary.html

Reply:
Thanks! I'll add that in!

Reply:
Added!